app/views/peer_to_peer_match_mailer/peer_to_peer_email.html.erb

```diff
@@ -0,0 +1 @@
+<%= content_tag(:div, @email_body) %>
```

Comment:
IIUC, `@email_body` is composed entirely of the message entered by the claimier on the [claim form](https://github.com/rubyforgood/mutual-aid/pull/730/files#diff-9aaded2bcd601dbec86b952e49d33003R14), right?

I suspect we'll need to add some more text around their message, explaining why the claimee is getting this email, what their next step is, how to get help etc.

Do you have thoughts on this @maebeale .

Also fine to merge as is, since this is easily updated.

Reply:
yup! that's my recollection also -- we have the body match the email body

agree on that text, and, the merging as is and refining afterward.